src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -2237,6 +2239,7 @@ export default defineComponent({
             // Rewind by the time-skip interval (in seconds)
             seekBySeconds(-defaultSkipInterval.value * player.getPlaybackRate(), false, true)
           }
+          if (canChapterJump(event, 'previous')) showOverlayControls()
```

Comment:
Same issue as with your playlist duration pull request, rather than integrating it into the existing code, you decided to copy-paste and duplicate code. Please move the `showOverlayControls()` call into the `if` block above.